staging/operator-lifecycle-manager/pkg/controller/registry/reconciler/grpc.go

```diff
@@ -146,6 +146,16 @@ func (c *GrpcRegistryReconciler) currentService(source grpcCatalogSourceDecorato
 	return service
 }
 
+func (c *GrpcRegistryReconciler) currentServiceAccount(source grpcCatalogSourceDecorator) *corev1.ServiceAccount {
+	serviceAccountName := source.ServiceAccount().GetName()
+	serviceAccount, err := c.Lister.CoreV1().ServiceAccountLister().ServiceAccounts(source.GetNamespace()).Get(serviceAccountName)
+	if err != nil {
+		logrus.WithField("service", serviceAccount).Debug("couldn't find service in cache")
```

Comment:
```suggestion
		logrus.WithField("serviceAccount", serviceAccount).Debug("couldn't find serviceAccount in cache")
```

Reply:
Yep - made this [same comment](https://github.com/openshift/operator-framework-olm/pull/294#discussion_r862861169) in the wrong PR. Should we just manually create a 4.10 PR so we can add that second commit?

Reply:
Yep, done in Created https://github.com/openshift/operator-framework-olm/pull/298